k8sinterface/k8sdiscovery.go

```diff
@@ -343,6 +343,10 @@ func StringToResourceGroup(str string) (string, string, string) {
 func updateResourceKind(resource string) string {
 	resource = strings.ToLower(resource)
 
+	if resource == "ingress" {
+		return "ingresses"
+	}
```

Comment:
@Daniel-GrunbergerCA maybe we should change this to be more generic, like:
`if !strings.HasSuffix(resource, "s"){
    return fmt.Sprintf("%ses", resource) // add 'es' at the end of a resource
}`
there are some other k8s resources that end in ss (StorageClass for .g) . we don't check any of them in any controls but it makes sense to be prepared..